packages/manager/apps/web/client/app/domain/webhosting/order/steps/duration/domain-webhosting-order-steps-duration.html

```diff
@@ -1,27 +1,24 @@
 <oui-step-form
-    data-header="{{:: 'domain_webhosting_order_duration_title' | translate }}"
+    data-submit-text="{{ :: 'ovh_product_offers_order_express_submit' | translate }}"
+    data-header="{{:: 'ovh_product_offers_pricing_header' | translate }}"
     data-on-focus="$ctrl.getAvailableDurations()"
-    data-on-submit="$ctrl.updateDuration()"
-    data-navigation="false"
-    data-editable="$ctrl.duration && $ctrl.hasManyDurations() && !$ctrl.stepper.loadingCheckout()"
+    data-on-submit="$ctrl.goToPromocodeFunnel()"
```

Comment:
Sorry if I'm missing something, but I don't find where this method is coming from.
If it's a mistake (something you forgot to commit) maybe it answer another question I have:
How is the duration sent to the express order ? (as below you removed the data-model on the select-picker and I have difficulties to understand how it is passed to the order funnel)

Reply:
No, you're not missing anything, I removed it but not in the html thks!